src/ui/components/NetworkMonitor/RequestTable.tsx

```diff
@@ -39,11 +41,19 @@ const RequestTable = ({
   const onSeek = (request: RequestSummary) => {
     trackEvent("net_monitor.seek_to_request");
     seek(request.point.point, request.point.time, true);
-    onRowSelect(request);
+    const syntheticEvent = new MouseEvent("click", {
+      bubbles: true,
+      cancelable: true,
+      view: window,
+      button: 0,
+    });
+    onRowSelect(request, syntheticEvent as unknown as React.MouseEvent<Element, MouseEvent>);
```

Comment:
This change seems unnecessary (the addition of the `event` prop)

Reply:
Yeah this was the strange stuff I was referring to. When I wrote the code without this, I was getting tdc errors, and it was only by adding this odd stuff that I could get it working.

I can take another look but this is beyond my understanding. Might need to reach out and ask for some help if/when I get the same tsc errors.

Reply:
Ugh, so embarrassing. I figured out why I ended up on that wild goose chase and reverted. What a dumb bit of code that was!

Reply:
No worries. We all do stuff like that. :)